x/rollapp/types/message_create_rollapp_test.go

```diff
@@ -3,84 +3,126 @@ package types
 import (
 	"testing"
 
-	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/dymensionxyz/dymension/v3/testutil/sample"
 	"github.com/stretchr/testify/require"
 )
 
 func TestMsgCreateRollapp_ValidateBasic(t *testing.T) {
+	defaultMetadata := TokenMetadata{
+		Description: "valid",
+		DenomUnits: []*DenomUnit{
+			{Denom: "uvalid", Exponent: 0},
+			{Denom: "valid", Exponent: 18},
+		},
+		Base:    "uvalid",
+		Display: "valid",
+		Name:    "valid",
+		Symbol:  "VALID",
+	}
+
 	seqDupAddr := sample.AccAddress()
 	tests := []struct {
 		name string
 		msg  MsgCreateRollapp
 		err  error
 	}{
+		{
+			name: "valid - full features",
+			msg: MsgCreateRollapp{
+				Creator:               sample.AccAddress(),
+				MaxSequencers:         1,
```

Comment:
maxsequencers value shouldnt be lower than the number of permissioned addresses introduced, otherwise it will fail once we merge the pr validating this